app/src/test/java/com/codesoom/assignment/application/ProductServiceTest.java

```diff
@@ -0,0 +1,149 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.domain.Product;
+import com.codesoom.assignment.exception.ResourceNotFoundException;
+import com.codesoom.assignment.repository.ProductJPARepository;
+import com.codesoom.assignment.service.ProductService;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+
+@DataJpaTest
+@DisplayName("ProductService 테스트")
+class ProductServiceTest {
+
+    private ProductJPARepository repository;
+    private ProductService service;
+
+    private final Long SIZE = 3L;
+    private final String TITLE = "proudct";
+    private final String MAKER = "maker";
+
+    public ProductServiceTest(){
+        repository = mock(ProductJPARepository.class);
+        service = new ProductService(repository);
+    }
+
+    Product getNewProduct(long number){
+        Product product = new Product(number , TITLE + number , MAKER + number , (int) number , null);
+        return product;
```

Comment:
```suggestion
        return new Product(number , TITLE + number , MAKER + number , (int) number , null);
```

한번 사용하고 버려질 변수는 굳이 만들지 않아도 됩니다!